.skills/SECURITY.md

```
@@ -0,0 +1,36 @@
+Run `npx skills use "https://github.com/getsentry/skills" --skill "security-review"` and follow the generated skill instructions now. Read its complete output, redirecting it to a temporary file first if necessary. Resolve relative paths from the supporting-files directory it provides.
+
+Identify exploitable security vulnerabilities in code. Report only HIGH CONFIDENCE findings—clear vulnerable patterns with attacker-controlled input.
+
+--- 
+
+Run `npx skills use "https://github.com/samber/cc-skills-golang" --skill "golang-security"` and follow the generated skill instructions now. Read its complete output, redirecting it to a temporary file first if necessary. Resolve relative paths from the supporting-files directory it provides.
+
+Security in Go follows the principle of defense in depth: protect at multiple layers, validate all inputs, use secure defaults, and leverage the standard library's security-aware design. Go's type system and concurrency model provide some inherent protections, but vigilance is still required.
+
+---
```

markdownlint-cli2 (0.23.1)

[warning] Horizontal rule style
Expected: --- ; Actual: ---

(MD035, hr-style)